<launch>
<node pkg="mir100_explore" type="explore" respawn="false" name="explore" output="screen">
  <param name="robot_base_frame" value="base_link"/>
  <param name="costmap_topic" value="map_irradiated"/>
  <param name="costmap_updates_topic" value="map_updates"/>
  <param name="visualize" value="true"/>
  <param name="planner_frequency" value="0.1"/>
  <param name="progress_timeout" value="30.0"/>
  <param name="potential_scale" value="0.001"/>
  <param name="orientation_scale" value="0"/>
  <param name="gain_scale" value="1"/>
  <param name="transform_tolerance" value="1"/>
  <param name="min_frontier_size" value="0.1"/>
</node>
</launch>
